pkg/agent/openflow/pipeline_windows.go

```diff
@@ -96,14 +44,16 @@ func (c *client) hostBridgeUplinkFlows(localSubnet net.IPNet, category cookie.Ca
 	if c.networkConfig.TrafficEncapMode.SupportsNoEncap() {
 		// If NoEncap is enabled, the reply packets from remote Pod can be forwarded to local Pod directly.
 		// by explicitly resubmitting them to ServiceHairpinTable and marking "macRewriteMark" at same time.
-		flows = append(flows, ClassifierTable.BuildFlow(priorityHigh).MatchProtocol(binding.ProtocolIP).
-			MatchInPort(config.UplinkOFPort).
-			MatchDstIPNet(localSubnet).
-			Action().LoadRegMark(FromUplinkRegMark).
-			Action().LoadRegMark(RewriteMACRegMark).
-			Action().GotoTable(ServiceHairpinTable.GetID()).
-			Cookie(c.cookieAllocator.Request(category).Raw()).
-			Done())
+		for _, localSubnet := range localSubnets {
+			flows = append(flows, ClassifierTable.BuildFlow(priorityHigh).MatchProtocol(binding.ProtocolIP).
```

Comment:
This is wrong. localSubnets includes one IPv4 subnet and one IPv6 subnet, setting protocol to `binding.ProtocolIP`. for both subnets will not work. I don't think windows or flexiable IPAM supports IPv6, why don't just add the IPv4 check back and keep the IPv6 TODO as is?

Reply:
Thanks, updated.